backend/steg_api.py

```diff
@@ -1 +1 @@
-from app import app
+from app import api
```

Comment:
Why did you change this? It didn't work until I changed it back. The app is imported here from the app folder.